docs/docs.md

```diff
@@ -25,49 +29,86 @@ Frontend
 cd frontend && yarn
 ```
 
-### Npm
+### npm
+
+Ensure you are using your installed version of Node.js (e.g., 20.16.0) by running `node --version`.
+
+Navigate to the root of this repository.
+
+Run `cd backend`.
+
+Run `npm i`.
+
+Run `mkdir out`.
+
+Run `cd ../frontend`.
+
+Run `npm i`.
+
+## Compile 
+
+### yarn
 
-Backend
 ```sh
-cd backend && npm i && mkdir out
+cd frontend && yarn run build && cd ../backend && yarn run compile
 ```
 
-Frontend
+### npm
+
+Run `cd ../frontend`.
+
+Run `npm run build`.
+
+Run `cd ../backend`.
+
+Run `npm run compile`.
+
+## Pack
+
+### yarn
+
 ```sh
-cd frontend && npm i
+cd backend && yarn run
 ```
 
-## Compile and pack
-### Yarn
-Compile:
- ```sh
-    cd frontend && yarn run build && cd ../backend && yarn run compile
- ```
-Pack:
-  ```sh
-    cd backend && yarn run
- ```
-
-### Npm
-Compile:
- ```sh
-    cd frontend && npm run build && cd ../backend && npm run compile
- ```
-Pack:
-  ```sh
-    cd backend && npm run pack
- ```
-
-
-This will generate an vsix that can be installed in azure data studio.
+### npm
+
+Run `cd ../backend`.
+
+Run `npm run pack`.
+
+Move `schema-visualization-0.9.3.vsix` to a desired location for installation as an extension in Azure Data Studio.
+
+## Install vsix File In Azure Data Studio
+
+Install Azure Data Studio (e.g., Azure Data Studio for User version 1.49.0).
+
+Open Azure Data Studio.
+
+Open Extensions.
+
+Click the ellipsis and Install from VSIX...
+
+Install file `Generator_Of_ERDs/backend/schema-visualization-0.9.3.vsix`.

```

Comment:
One last thing: "Generator_Of_ERDs" seem to be the name of your fork.

Also the 0.9.3 is the version number that will vary each time

Reply:
@R0tenur , I generalized appropriately.